skills/lark-okr/SKILL.md

```
@@ -14,30 +14,87 @@ metadata:
+lark-cli okr +indicator-update \
+  --level key-result \
+  --id 7000000000000000003 \
+  --value 75 \
+  --as user
+```
+
+分数和进度不要混用：用户说“进度”“完成度”“当前做到 75%”时，通常是在改量化指标或写进展记录，不是在改 `score`。只有明确要求修改
+OKR 分数/评分/打分时，才使用 [`+patch --score`](references/lark-okr-patch.md)；`score` 取值是 0-1，最多一位小数。
+
+进度判断规则：用户说“进度”“完成度”时，先判断是否是量化数字。数字进度通常对应量化指标；不可量化文本对应进展记录。需要修改指标单位时看 [
+`lark-okr-indicators.md`](references/lark-okr-indicators.md)
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)